index.bs

```diff
@@ -550,6 +550,14 @@ representation of an object judged to be maximally useful and informative.
   </code></pre>
 </div>
 
+<h3 id="reporting-warnings">Reporting warnings to the console</h3>
+
+To <dfn export>report a warning to the console</dfn> given a generic description of a warning
+|description|, implementations must perform:
```

Comment:
Nit: for stylistic consistency with other specs I would favor the construct
`implementations must run these steps: ...` over
 `implementations must perform: ...`

For example, in the DOM standard at https://dom.spec.whatwg.org the text "must run these steps" currently occurs 73 times wheras "must perform" does not seem to appear at all.